src/plots/VolumeVTK9/VolumeAttributes.h

```diff
@@ -38,11 +38,11 @@ class VolumeAttributes : public AttributeSubject
     };
     enum ResampleType
     {
+        NoResampling,
```

Comment:
Was this ordering change necessary?

Reply:
The change actually came from running xml2atts.  My guess is that Alan manually changed the ordering in the header, but not the xml file. Think I should update the xml file instead, to preserve the ordering in the atts file?'t 
I don't know what the effect of the current change would have.